plugins/inputs/sflow/sflow_version_5.txt

```diff
@@ -0,0 +1,2803 @@
+
+
+
+
+
+
+sFlow.org                                                    Peter Phaal
+http://www.sFlow.org/                                        InMon Corp.
+info@sflow.org
+
+                                                             Marc Lavine
+                                                        Foundry Networks
+
+                                                               July 2004
+
+
+                            sFlow Version 5
+
+
+
+Copyright Notice
+
+Copyright (C) sFlow.org (2004). All Rights Reserved.
```

Comment:
It doesn't look like we are allowed to copy this, let's link to this in the documentation but not include it.